bin/rolling_index

```diff
@@ -2,27 +2,34 @@
 
 require_relative '../config/environment'
 require 'daemons'
+require 'parallel'
 
 QUERY = { q: '*:*', sort: 'timestamp asc', fl: 'id', rows: '500' }
 
 Daemons.run_proc('rolling index') do
   loop do
     solr_conn = RSolr.connect(timeout: 120, open_timeout: 120, url: Settings.solrizer_url)
     response = solr_conn.get 'select', params: QUERY
-    response['response']['docs'].each do |doc|
+    solr_docs = Parallel.map(response['response']['docs'], in_processes: 8) do |doc|
       identifier = doc['id'].scrub('')
       # Occasionally, we've seen invalid bytes in the identifier, so try to catch those:
       Honeybadger.notify("Identifier isn't valid utf-8", { identifier: identifier, bytes: identifier.unpack('C*') }) unless doc['id'].valid_encoding?
       begin
+        # This returns a Solr doc instance
         Indexer.load_and_index(solr: solr_conn, identifier: identifier)
```

Comment:
it looks from the `Indexer.load_and_index` ultimately does a `solr_conn.add` underneath, so i'm wondering if that means this the `solr_conn.add` that was added in this PR is redundant?  https://github.com/sul-dlss/dor_indexing_app/blob/ad5ee6221adc0f81c2a906aded54b6f6bc73fd1a/app/services/indexer.rb#L35-L53

i couldn't quite tell (based on title and description) whether the goal of this PR was only to gain speed via the parallelization change, or to also gain speed via reduction of the number of solr doc adds.  if the latter, and if i'm understanding the code correctly, it seems like we'd maybe want to refactor `Indexer` to have a method that only returns the solr doc instead of also adding it, or to replace the `Indexer.load_and_index` call here with e.g. `DocumentBuilder.for(model: Dor::Services::Client.object(identifier).find).to_solr` (and if going that route, to maybe add similar `Honeybadger.context` setting to what would be lost if `Indexer` isn't used).